django_comparison_dashboard/graphs.py

```diff
@@ -191,6 +192,21 @@ def sankey(data, filter_set: SankeyGraphFilterSet):
 
         value.append(flow["value"])
 
+    node_colors = [
+        f'rgba({int(color_dict[label][1:3], 16)}, {int(color_dict[label][3:5], 16)}, {int(color_dict[label][5:7], 16)}, 0.75)'
```

Comment:
I would create a small helper function, like this: https://community.plotly.com/t/scatter-plot-fill-with-color-how-to-set-opacity-of-fill/29591/2
in order to implement conversion from hex to rgb - this increases readability and adheres to DRY concept.

Reply:
Thank you for the remark, I added it in a new commit.